### ISSUE TITLE
Inpainting as object removal

### ISSUE THREAD

**Issue by himmetozcan** [runwayml/stable-diffusion#21]
First of all thanks for the great work. I have a question related to the inpainting with SD. If I want to remove an object completely from the scene, what text prompt should I use? An empty text, or some text describing the background? Thanks!

**Comment by Dima-369** [comment]
I find describing the background works well, but leaving the prompt often works surprisingly well too.

Just try it out yourself, it depends on the image.

**Comment by Sanster** [comment]
Or you can just use [Lama Cleaner](https://github.com/Sanster/lama-cleaner)

**Comment by himmetozcan** [comment]
I have checked the Lama Cleaner. In that, they also use SD-1.5 which uses an empty string for the text prompt. I must say that ckpt v1.5 of runwayml inpainting is not removing the object all the time. Sometimes it just puts a random object. I have also tried [stable-diffusion-2](https://huggingface.co/stabilityai/stable-diffusion-2). It looks like working better in terms of removing the object. Thanks for the comments.
